client-java/instrumentation/src/main/resources/keepInstrumentationList.txt

```diff
@@ -1,4 +1,3 @@
 //list of classes/packages to include regardless of skipList.
 //this is useful when you want to ban a domain org.foo but still allow
 //some of its subpackages, like org.foo.bar
-
```

Comment:
any reason for modyfing this file?

Reply:
Did some experiments, previous commit captured the changes. In this reverted it back to the original version.